test/e2e/cert_rotation.go

```diff
@@ -72,40 +57,33 @@ func certRotation(ctx context.Context, inputGetter func() CertRotationInput) {
 	}
 	for _, secretName := range secretList {
 		err := clientSet.CoreV1().Secrets(ironicNamespace).Delete(ctx, secretName, metav1.DeleteOptions{})
```

Comment:
Probably we should add step to confirm the secrets do not exist before checking they are recreated

Reply:
Done